students/Scott_Hancock/fizz_buzz/fizzbuzz.py

```diff
@@ -0,0 +1,12 @@
+# prints the FizzBuzz problem
+for i in range(1, 101):
+    if (i % 3) == 0 and (i % 5) == 0:
+        print("FizzBuzz")
+    elif i % 3 == 0:
+        print("Fizz")
+    elif i % 5 == 0:
+        print("Buzz")
+    else:
+        print(i)
+
```

Comment:
Looks good! For an additional challenge, see if you can figure out how to print the fizzbuzz for numbers divisible for both 3 and 5 without an explicit conditional for both!